tests/snuba/test_snuba.py

```diff
@@ -44,3 +47,24 @@ def test_fail(self):
                 filter_keys={'project_id': [100]},
                 groupby=[")("],
             )
+
+    @patch('django.utils.timezone.now')
+    def test_get_project_issues(self, mock_time):
+        now = datetime(2018, 1, 1, 0, 0, 0, tzinfo=pytz.utc)
+        mock_time.return_value = now
+        assert snuba.get_project_issues([self.project]) == []
+
+        GroupHash.objects.create(
+            project=self.project,
+            group=self.group,
+            hash='a' * 32
+        )
+        assert snuba.get_project_issues([self.project]) == [(1, [('aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa', None)])]
+
+        GroupHashTombstone.tombstone_groups(self.project.id, [self.group.id])
+        GroupHash.objects.create(
+            project=self.project,
+            group=self.group,
+            hash='a' * 32
+        )
+        assert snuba.get_project_issues([self.project]) == [(1, [('aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa', now)])]
```

Comment:
I'd feel more comfortable with a couple more tests. I guess there are also tests on the snuba side that test that tombstoned issues are not returned, but it would be good to see a full e2e test that demonstrates issue creation, deletion, and resurrection with a new event?

Reply:
Definitely agree, for some reason I thought I did that but I guess it was manual. Working on it now.